web/src/components/hero.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BackgroundImage from 'gatsby-background-image';
+
+export default function Hero({ image }) {
+    const {
+        childImageSharp: { fluid },
+    } = image;
+
+    return (
+        <BackgroundImage className="w-full min-h-screen" fluid={fluid}>
```

Comment:
Can you add `Tag="section"` so we get an element that's more semantic than a `<div>`.

Also I think we can do `flex justify-center items-center` for centering the heading. Then you should be able to drop all of the positioning and transform styling from the `<h1>`.

Also, I'm not sure `min-h-screen` is going to be right. The designs show height at `720`. That might be full screen, I'm not sure. We could pass `h-[820] md:h-[720px]` (or it's rem equivalent).

Reply:
@asasmith when I did the walk through with Josh, he said they wanted the item to take full height of the screen.